src/ezdxf/entities/dxfgfx.py

```diff
@@ -238,8 +238,22 @@ def rgb(self) -> Optional[clr.RGB]:
 
     @rgb.setter
     def rgb(self, rgb: clr.RGB) -> None:
-        """Set RGB true color as (r, g , b) tuple e.g. (12, 34, 56)."""
-        self.dxf.set("true_color", clr.rgb2int(rgb))
+        """
+        Set RGB true color as (r, g , b) tuple e.g. (12, 34, 56).
+        Also allows setting/resetting rgb to None.
+        """
+        if rgb is None:
```

Comment:
This case should raise a TypeError, None is not a valid input or revert everything because the previous version raised the TypeError automatically.